internal/internal_workflow_test.go

```diff
@@ -757,6 +758,32 @@ func (s *WorkflowUnitTest) Test_CorruptedSignalOnClosedChannelWorkflow_Receive_S
 	s.EqualValues(0, len(result))
 }
 
+func bufferedChanWorkflowTest(ctx Context) error {
```

Comment:
Can we add a parameter to this workflow that includes buffer size? That way you can easily write another test for buffer size greater than one.

Reply:
Yeah I can change the bufferSize to a parameter.